### PR TITLE
Ft/tosho/register and login seller endpoints

### PR DESCRIPTION
**Pull request by cheppymk**

### CODE REVIEW

File: Shipfinity.DTOs/SellerDTO's/SellerLoginResponseDto.cs [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+using System.Threading.Tasks;
+
+namespace Shipfinity.DTOs.SellerDTO_s
+{
+    public class SellerLoginResponseDto
+    {
+        public int Id { get; set; }
+        public string Username { get; set; }
+        public string Email { get; set; }
+        public string Name { get; set; }
```
Comment (KristijanJankuloski) [inline Shipfinity.DTOs/SellerDTO's/SellerLoginResponseDto.cs:14]: add the role aswell `public string Role { get; set; }`

File: Shipfinity.Api/appsettings.json [hunk 2]
```diff
@@ -7,7 +7,7 @@
   },
   "AllowedHosts": "*",
   "ConnectionStrings": {
-    "ShipfinityDbString": "Data Source=localhost\\SQLEXPRESS;Database=ShipfinityDb;Integrated Security=True;Connect Timeout=30;Encrypt=False;Trust Server Certificate=False;Application Intent=ReadWrite;Multi Subnet Failover=False"
+    "ShipfinityDbString": "Data Source=(localdb)\\LocalDBDemo;Database=ShipfinityDbTest;Integrated Security=True;Connect Timeout=30;Encrypt=False;Trust Server Certificate=False;Application Intent=ReadWrite;Multi Subnet Failover=False"
```
Comment (KristijanJankuloski) [inline Shipfinity.Api/appsettings.json:10]: revert changes to connection string to `"Data Source=localhost\\SQLEXPRESS;Database=ShipfinityDb;Integrated Security=True;Connect Timeout=30;Encrypt=False;Trust Server Certificate=False;Application Intent=ReadWrite;Multi Subnet Failover=False"`

**Review by KristijanJankuloski — CHANGES_REQUESTED** [review]

File: Shipfinity.Mappers/SelllerMappers.cs [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+        }
+
+        public static SellerLoginResponseDto ToLoginResponse(this Seller seller, string token)
+        {
+            return new SellerLoginResponseDto
+            {
+                Id = seller.Id,
+                Username = seller.Username,
+                Email = seller.Email,
+                Token = token
```
Comment (KristijanJankuloski) [inline Shipfinity.Mappers/SelllerMappers.cs:34]: Map role aswell

**Review by KristijanJankuloski — CHANGES_REQUESTED** [review]

**Review by Aleksandar1105 — APPROVED** [review]

File: Shipfinity.DTOs/SellerDTO's/SellerLoginResponseDto.cs (hunk 1 above)
Comment (cheppymk) [reply Shipfinity.DTOs/SellerDTO's/SellerLoginResponseDto.cs:14]: done

File: Shipfinity.Api/appsettings.json (hunk 2 above)
Comment (cheppymk) [reply Shipfinity.Api/appsettings.json]: done

**Pull request merged**